metadata-ingestion/src/datahub/utilities/sql_parser.py

```diff
@@ -5,15 +5,15 @@
 from abc import ABCMeta, abstractmethod
 from typing import List, Set
 
+import sqlparse
+from networkx import DiGraph
+from sqllineage.core import LineageAnalyzer
 from sqllineage.core.holders import Column, SQLLineageHolder
 
+import datahub.utilities.sqllineage_patch
+
 try:
-    import sqlparse
-    from networkx import DiGraph
     from sql_metadata import Parser as MetadataSQLParser
-    from sqllineage.core import LineageAnalyzer
-
-    import datahub.utilities.sqllineage_patch
 except ImportError:
```

Comment:
These imports are updated to ensure that no imports needed for DefaultSQLParser (sqllineage) fail just because sql_metadata import gives an error. Earlier sequence forced to install "sql-metadata" pypi library even default parser is used.